src/components/Header/Header.css

```diff
@@ -0,0 +1,55 @@
+.clear-fix {
+  overflow: auto;
+}
+
+.welcome-message {
+  font-style: italic;
+  font-size: 18px;
+  color: #f3f3f3;
+  line-height: 28px;
+}
+
+#skills-h3 {
+  color: #f5ae23;
+  display: none;
+}
+
+#header {
+  background-color: #484848;
+}
+
+.flex-column {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+  padding: 10px;
+}
+
+.center-content {
+  padding: 2.5% 5%;
+}
+
+.biopic {
+  float: left;
+  padding: 10px;
+  width: 200px;
+  display: none;
+}
+
+@media only screen and (max-width: 900px) {
+  .biopic {
+    width: 175px;
+  }
+}
+
+@media only screen and (max-width: 750px) {
+  .biopic {
+    width: 150px;
+  }
+}
+
+@media only screen and (min-width: 750px) {
```

Comment:
The media query here overlapped with max-width above. Better set the boundary with +-1 e.g.
`min-width: 751px`

So you can use the same selector `.biopic`